integration/testdata/init/hello/deployment.yaml.expected

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: dockerfile-image
```

Comment:
What happens if there are 2 Dockerfiles. Would the `spec.containers.name` name still be `dockerfile-image`?

If yes, the `GenerateMaifests` should only generate deployment.yaml and service.yaml if unresolved dockerfiles is 1.

Reply:
if there are two Dockerfiles, they're generated based on path, so for the microservices example this would be `leeroy-web/deployment.yaml` and `leeroy-app/deployment.yaml`. I wrote a test for this, but because the `--force` flag is broken right now the test can't pass, so I commented it out with a TODO until that's fixed.

Reply:
yes the path will be different but will the generated name, label and container image names `dockerfile-image` deployment spec be different?

Reply:
yeah you got it in your other comment, all the resource names are specific to the path/file name so they'll be different.